### PR TITLE
文档错误

### PR DESCRIPTION
**Pull request by lishihao007**
### First-time contributors' checklist 
yes
- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.
yes
### What is changed, added or deleted? (Required)
修改

修改date和datetime范围说明，实测 0000-01-01 到 9999-12-31 ，官方写的 1000-01-01 到 9999-12-31
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x ] master (the latest development version)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lishihao007** [comment]
第一次提交PR，请问目前什么状态，有什么问题

**Comment by lishihao007** [comment]
现在到哪一步了

**Review by tangenta — APPROVED** [review]
Confirmed that the old description is wrong:

https://github.com/pingcap/tidb/blob/94e30df8e2d8ba2a1a26f153f40067ba3acd78eb/types/time.go#L2060-L2070

**Comment by lishihao007** [comment]
还没有改掉吗

File: data-type-numeric.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ TiDB 支持 MySQL 所有的定点类型，包括 `DECIMAL`、`NUMERIC`，完整
 
 #### `DECIMAL` 类型
 
-定点数，别名为 `NUMERIC`。M 是小数位数（精度）的总数，D 是小数点（标度）后面的位数。小数点和 `-`（负数）符号不包括在 M 中。如果 D 是 0，则值没有小数点或分数部分。如果 D 被省略，默认是 0。如果 M 被省略，默认是 10。
+定点数，别名为 `NUMERIC`。M 是数值位数（精度）的总数（整数位数+小数位数），D 是小数点（标度）后面的位数。小数点和 `-`（负数）符号不包括在 M 中。如果 D 是 0，则值没有小数点或分数部分。如果 D 被省略，默认是 0。如果 M 被省略，默认是 10。
```
Comment (TomShawn) [inline data-type-numeric.md:193]:
```suggestion
定点数，别名为 `NUMERIC`。M 是数值位数（精度）的总数（整数位数 + 小数位数），D 是小数点（标度）后面的位数。小数点和 `-`（负数）符号不包括在 M 中。如果 D 是 0，则值没有小数点或分数部分。如果 D 被省略，默认是 0。如果 M 被省略，默认是 10。
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**